Source/Core/Editor/UI/Views/ManageGroupsPanel.uxml

```diff
@@ -1,6 +1,6 @@
 <ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="True">
-    <Style src="project://database/Packages/co.mindport.vrbuilder.core/Source/Core/Editor/UI/Views/GroupListStyles.uss?fileID=7433441132597879392&amp;guid=16329c9ae374d64418149d370315f789&amp;type=3#GroupListStyles" />
-    <Style src="project://database/Packages/co.mindport.vrbuilder.core/Source/Core/Editor/UI/Views/ManageGroupPanelStyles.uss?fileID=7433441132597879392&amp;guid=d0080977fd56a49418b05a64dbe41285&amp;type=3#ManageGroupPanelStyles" />
+    <Style src="project://database/Assets/MindPort/VR%20Builder/Core/Source/Core/Editor/UI/Views/GroupListStyles.uss?fileID=7433441132597879392&amp;guid=16329c9ae374d64418149d370315f789&amp;type=3#GroupListStyles" />
+    <Style src="project://database/Assets/MindPort/VR%20Builder/Core/Source/Core/Editor/UI/Views/ManageGroupPanelStyles.uss?fileID=7433441132597879392&amp;guid=d0080977fd56a49418b05a64dbe41285&amp;type=3#ManageGroupPanelStyles" />
```

Comment:
How come the path changes here? Would this cause problems?

Reply:
Hm good catch I have to test this its basically what I worte you on slack